plugins/pagerduty-incidents/src/components/PagerDutyIncidents.tsx

```diff
@@ -72,183 +51,24 @@ const PagerDutyIncidents: React.FC<PagerDutyIncidentsProps> = ({
     [entityYaml?.info]
   );
 
-  useEffect(() => {
-    const fetchService = async (): Promise<void> => {
-      setIsLoading(true);
-      setErrorMessage("");
-      if (!pdId || pdType !== "SERVICE") {
-        setErrorMessage(
-          "This entity is not associated with a PagerDuty service"
-        );
-        setIsLoading(false);
-        return;
-      }
-      try {
-        const url = `https://api.pagerduty.com/services/${pdId}`;
```

Comment:
During the service discovery phase, if the plugin can't communicate with the API, the dropdown just gets stuck on "Loading..." and there's no visual feedback, except for in the JavaScript console. Can we provide visual feedback in the Plugin that shows that the API hook failed?